### PR TITLE
Processing FlowRuleEvents only for devices we're master of and flows …

### PR DESCRIPTION
**Pull request by teone**
…that we manage

### CODE REVIEW

File: impl/src/main/java/org/opencord/olt/impl/OltFlowService.java [hunk 1]
```diff
@@ -1435,9 +1436,22 @@ public int hashCode() {
...
         @Override
         public void event(FlowRuleEvent event) {
+
+            if (!appId.equals(event.subject().appId())) {
+                return;
+            }
+
+            if (!oltDeviceService.isLocalLeader(event.subject().deviceId())) {
+                if (log.isTraceEnabled()) {
+                    log.trace("ignoring meter event {} " +
```
Comment (Andrea-Campanella) [inline impl/src/main/java/org/opencord/olt/impl/OltFlowService.java]: change the log to "ignoring flow event ...."

File: impl/src/main/java/org/opencord/olt/impl/OltFlowService.java [hunk 2]
```diff
@@ -1455,7 +1469,7 @@ public void event(FlowRuleEvent event) {
             }
         }
 
-        private void updateCpStatus(FlowRuleEvent.Type type, ConnectPoint cp, FlowRule flowRule) {
+        public void updateCpStatus(FlowRuleEvent.Type type, ConnectPoint cp, FlowRule flowRule) {
```
Comment (Andrea-Campanella) [inline impl/src/main/java/org/opencord/olt/impl/OltFlowService.java]: why public ? probably protected is enough.

Comment (teone) [reply impl/src/main/java/org/opencord/olt/impl/OltFlowService.java]: done

File: impl/src/main/java/org/opencord/olt/impl/OltFlowService.java (hunk 1 above)
Comment (teone) [reply impl/src/main/java/org/opencord/olt/impl/OltFlowService.java]: done

**Pull request merged**